index.html

```diff
@@ -4,15 +4,11 @@
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Speedometer 2.1</title>
     <link rel="stylesheet" href="resources/main.css">
-    <script src="resources/main.js" defer></script>
-    <script src="resources/benchmark-runner.js" defer></script>
-    <script src="resources/benchmark-report.js" defer></script>
-    <script src="../resources/statistics.js" defer></script>
-    <script src="resources/tests.js" defer></script>
+    <script src="resources/main.mjs" type="module"></script>
 </head>
 <body>
 <main>
-    <a id="logo-link" href="javascript:showHome()"><img id="logo" src="resources/logo.png"></a>
+    <img id="logo" src="resources/logo.png">
```

Comment:
This will make it less keyboard accessible. We need to either make the logo itself focusable or add back anchor element.